src/core/client/framework/lib/validation.tsx

```diff
@@ -153,6 +154,15 @@ export const validateEqualPasswords = createValidator(
   PASSWORDS_DO_NOT_MATCH()
 );
 
+/**
+ * validateEventSelection is a Validator that checks for a valid combination of
+ * event selections for webhook endpoints.
+ */
+export const validateEventSelection = createValidator(
```

Comment:
Can we indicate that this is specific to webhooks in the name of this validator?